xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -1797,8 +1985,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reserves space on the Guid heap for a GUID.</summary>
+        <returns>Handle to the reserved Guid and a <see cref="T:System.Reflection.Metadata.Blob" /> representing the GUID blob as stored on the heap.</returns>
```

Comment:
*and* a blob? or "a handle to the reserved Guid in a blob representing the GUID blob as stored on the heap"?